databricks_cli/unity_catalog/ext_loc_cli.py

```diff
@@ -163,17 +203,30 @@ def delete_location_cli(api_client, name, force):
 @click.option('--cred-aws-iam-role', default=None,
               help='An aws role to validate')
 @click.option('--cred-az-directory-id', default=None,
-              help='An Azure directory id to validate')
+              help='An Azure Service Principal directory id to validate')
 @click.option('--cred-az-application-id', default=None,
-              help='An Azure application id to validate')
+              help='An Azure Service Principal application id to validate')
 @click.option('--cred-az-client-secret', default=None,
-              help='An Azure directory id to validate')
+              help='An Azure Service Principal directory id to validate')
+@click.option('--cred-az-mi-access-connector-id', default=None,
+              help='An Azure Managed Identity access connector id to validate')
+@click.option('--cred-az-mi-id', default=None,
+              help='An Azure Managed Identity id to validate')
+@click.option('--cred-gcp-sak-email', default=None,
+              help='A GCP Service Account Key email to validate')
+@click.option('--cred-gcp-sak-private-key-id', default=None,
+              help='A GCP Service Account Key private key ID to validate')
+@click.option('--cred-gcp-sak-private-key', default=None,
+              help='A GCP Service Account Key private key to validate')
 @debug_option
 @profile_option
 @eat_exceptions
 @provide_api_client
 def validate_location_cli(api_client, name, url, cred_name, cred_aws_iam_role, cred_az_directory_id,
-                          cred_az_application_id, cred_az_client_secret):
+                          cred_az_application_id, cred_az_client_secret,
+                          cred_az_mi_access_connector_id,
+                          cred_az_mi_id, cred_gcp_sak_email, cred_gcp_sak_private_key_id,
+                          cred_gcp_sak_private_key):
```

Comment:
No biggie, but I wonder if this CLI should be `validate_storage_credential`. It still requires a `url`, of course, but the CLI name would at least match the API endpoint.  Eh, just a thought...

Reply:
Yeah, I am not sure why it's named differently, but unfortunately, renaming a command is more trouble than it's worth. If we want to, it should be in a different PR with proper deprecation patterns.

Reply:
Agreed.